perllib/FixMyStreet/App/Controller/Admin/Bodies.pm

```diff
@@ -283,6 +283,13 @@ sub update_contact : Private {
             $contact->unset_extra_metadata($_);
         }
     }
+    foreach (qw(title_hint detail_hint)) {
+        if ( my $value = $c->get_param($_) ) {
+            $contact->set_extra_metadata( $_ => $value );
```

Comment:
This allows you to hide the hint entirely by entering a string containing only whitespace - is that desirable? I'm not sure if it is so we should maybe trim the string before the test.

Reply:
I guess someone could decide they didn't want a hint to be shown? Hmm, no probably want to always have something, will do.